### PR TITLE
Update for new devices

### PR DESCRIPTION
**Pull request by stevemoser**

### CODE REVIEW

File: Source/Device.swift [hunk 1]
```diff
@@ -83,7 +85,7 @@ public class Device {
             "iPad4,4", "iPad4,5", "iPad4,6",
             "iPad4,7", "iPad4,8", "iPad4,9",
             "iPad5,1", "iPad5,2",
-            "iPad6,7", "iPad6,8":                           return Type.iPad
+            "iPad6,3", "iPad6,4", "iPad6,7", "iPad6,8":                           return Type.iPad
```
Comment (Ekhoo) [inline Source/Device.swift]: Can you plz keep the indentation.

Comment (stevemoser) [inline Source/Device.swift]: @Ekhoo Good call, done.

**Comment by stevemoser** [comment]
@Ekhoo Is there anything else I need to change for this PR to be accepted?

**Comment by Ekhoo** [comment]
It's perfect, I will release a new version as soon as possible

**Comment by Ekhoo** [comment]
Thank you :+1:

**Comment by Ekhoo** [comment]
Device `1.0.0` is available

**Pull request merged**